src/app/pages/test-report/components/toolbar/toolbar.scss

```diff
@@ -1,14 +1,16 @@
 :host {
-  #wrapper {
-    height: 68px;
-    padding: 12px 24px;
+  .toolbar-wrapper {
+    height: 56px;
+    padding: 6px 24px;
     background: var(--mes-white);
+    //padding-top: 6px;
+    //padding-bottom: 4px;
```

Comment:
please remove

Reply:
already commented that i'd remove that :D